components/script/layout_dom/node.rs

```diff
@@ -387,6 +387,11 @@ impl<'dom> ThreadSafeLayoutNode<'dom> for ServoThreadSafeLayoutNode<'dom> {
         this.svg_data()
     }
 
+    fn svg_image_data(&self) -> Option<SVGImageData> {
```

Comment:
This isn't actually called anywhere right now. Are you planning to add that in a followup?

Reply:
Actually no plans to implement it.
I just copy-pasted all stuff from SVGSVGElement to make it the same as much as possible (BTW svg_data also doesn't use anywhere)

Reply:
Ok, then let's remove the attribute parsing and new traits.

Reply:
Done